src/custom-elements.js

```diff
@@ -465,7 +476,7 @@ let Deferred;
       visitedNodes.add(element);
 
       /** @type {?CustomElementDefinition} */
-      const definition = this._definitions.get(element.localName);
+      const definition = getDefinition(this._definitions, element);
```

Comment:
Getting the definition for an element is now more complicated than it used to be -- we need to check if the tagName is a custom element but we also need to check if the `is=""` is a custom element. Since that logic is reused in several places, I extracted this out into the `getDefinition` function